src/unix/thread.c

```diff
@@ -102,10 +103,87 @@ int uv_thread_create(uv_thread_t *tid, void (*entry)(void *arg), void *arg) {
 }
 
 
+#if defined(__APPLE__) && defined(__MACH__) || defined(_AIX)
+int uv_thread_cpumasksize(void) {
+  return -ENOTSUP;
+}
+
+int uv_thread_setaffinity(uv_thread_t* tid,
+                          char* cpumask,
+                          char* oldmask,
+                          size_t mask_size) {
+  return -ENOTSUP;
+}
+
+
+int uv_thread_getaffinity(uv_thread_t* tid,
+                          char* cpumask,
+                          size_t mask_size) {
+  return -ENOTSUP;
+}
+
+#else /* !((defined(__APPLE__) && defined(__MACH__)) || defined(_AIX)) */
+
+int uv_thread_cpumasksize(void) {
+  return CPU_SETSIZE;
+}
+
+int uv_thread_setaffinity(uv_thread_t* tid,
+                          char* cpumask,
+                          char* oldmask,
+                          size_t mask_size) {
+  int i;
+  int r;
+  cpu_set_t cpuset;
+  int cpumasksize;
+  
+  cpumasksize = uv_thread_cpumasksize();
+  assert(mask_size >= (size_t)cpumasksize);
+
+  if (oldmask) {
+    r = uv_thread_getaffinity(tid, oldmask, mask_size);
+    if (!r)
+      return r;
+  }
+
+  CPU_ZERO(&cpuset);
+  for (i = 0;  i < cpumasksize;  i++)
```

Comment:
Too many spaces and there's an indent error two lines below.  Same issue in the next function.